tests/modules/submissions/test_tus_creation.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name,missing-docstring
+import os
+import shutil
+
+
+def test_create_submission_from_tus(db, researcher_1):
+
+    from app.modules.submissions.models import Submission
+    from tests.modules.sightings.resources.utils import (
+        prep_tus_dir,
+        get_transaction_id,
+    )  # recycle
+
+    tid = get_transaction_id()  # '11111111-1111-1111-1111-111111111111'
+
+    # first no such dir exists
+    try:
+        sub = Submission.create_submission_from_tus('PYTEST', researcher_1, tid)
+    except OSError:  # this is expected failure behavior here
+        pass
+
+    # now with a file dir+files but ask for wrong one
+    tid, valid_file = prep_tus_dir()
+
+    try:
+        sub = Submission.create_submission_from_tus(
+            'PYTEST', researcher_1, tid, paths={'fail.jpg'}
+        )
+    except OSError:  # this is expected failure behavior here
+        pass
+
+    # test with explicit paths (should succeed)
+    sub = Submission.create_submission_from_tus(
+        'PYTEST', researcher_1, tid, paths={valid_file}
+    )
+    assert len(sub.assets) == 1
+    assert sub.assets[0].path == valid_file
+    if os.path.exists(sub.get_absolute_path()):
+        shutil.rmtree(sub.get_absolute_path())
+    sub.delete()
+    return
```

Comment:
last part of the test not executed, intentional or oversight?

Reply:
oversight!  you are correct.  `- return`